pyvrp/cpp/TimeWindowSegment.h

```diff
@@ -66,14 +70,22 @@ class TimeWindowSegment
     [[nodiscard]] Duration duration() const;
 
     /**
-     * Returns the total time warp on this route segment.
+     * Returns the time warp on this route segment. This is all time warp
+     * incurred due to time windows, service durations, travel, and client
+     * release times.
+     *
+     * .. note::
+     *
+     *    This method does not know about time warp incurred in other places,
+     *    e.g. due to route-level constraints on maximum duration. Such time
+     *    warp is tracked outside of :class:`~pyvrp._pyvrp.TimeWindowSegment`.
      *
      * Returns
      * -------
      * int
      *     Total time warp on this route segment.
      */
-    [[nodiscard]] inline Duration totalTimeWarp() const;
+    [[nodiscard]] inline Duration timeWarp() const;
```

Comment:
The old name hasn't made sense in a while, so I figured I'd update it as I was here writing additional documentation anyway.